src/main/scala/scala/swing/ListView.scala

```diff
@@ -194,16 +194,14 @@ class ListView[A] extends Component {
      * The indices of the currently selected items.
      */
     object indices extends Indices(peer.getSelectedIndices) {
-      def -=(n: Int): this.type = { peer.removeSelectionInterval(n,n); this }
-      def +=(n: Int): this.type = { peer.addSelectionInterval(n,n); this }
+      def subtractOne(n: Int): this.type = { peer.removeSelectionInterval(n,n); this }
+      def addOne(n: Int): this.type = { peer.addSelectionInterval(n,n); this }
     }
 
     /**
      * The currently selected items.
      */
-    object items extends scala.collection.SeqProxy[A] {
-      def self = peer.getSelectedValues.map(_.asInstanceOf[A])
-    }
+    val items: Seq[A] = peer.getSelectedValuesList.asScala
```

Comment:
This is not equivalent, as it becomes an immutable seq. The idea was (as far as I remember) that you can modify the selection by, for example, adding elements to `items`.

Reply:
Tricky.

Reply:
Actually I was wrong. `SeqProxy` did _not_ allow the sequence to be modified.